nautobot/docs/user-guide/administration/installation/services.md

```diff
@@ -320,24 +370,26 @@ Please see [Computed fields with fallback value that is unicode results in Opera
 
 When serving Nautobot directly from uWSGI on RedHat or CentOS there may be a problem rendering .svg images to include the Nautobot logo. On the RedHat based operating systems there is no file `/etc/mime.types` by default, unfortunately, uWSGI looks for this file to serve static files (see [Serving static files with uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/StaticFiles.html#mime-types)). To work around this copy the file `/etc/mime.types` from a known good system for example an Ubuntu/Debian system or even the Nautobot container to /opt/nautobot/mime.types. Then add the following line to your `uwsgi.ini` file and restart the Nautobot services:
 
-```no-highlight
+```no-highlight title="Add MIME file settings to uwsgi.ini"
 mime-file = /opt/nautobot/mime.types
 ```
 
 Alternatively, host Nautobot behind Nginx as instructed in [HTTP server setup](http-server.md).
 
 ### Test Redis Connectivity
 
```

Comment:
Would be good to have some text here about when/why you'd *need* to test the Redis connectivity. Not blocking for this PR.